### DISCUSSION TITLE
LazyVim: Is there a way to have it load when my Second Brain is opened in nvim?

### DISCUSSION THREAD

**Discussion by josephcrawfordSRH** [epwalsh/obsidian.nvim#428] [Q&A] [answered]
I know I can have lazy load the plugin when a markdown file is opened but when I go to my second brain and open nvim I am not necessarily opening a markdown file, I would like to do :ObsidianNew but cannot unless I open a markdown file to load the extension.

The tree knows what directory I am in so there must be an event that I could bind to in order to load this I am just so new that I am unsure what or how I would do that.

**Comment by akthe-at** [comment, marked as answer]
```
    cmd = {
      "ObsidianOpen",
      "ObsidianQuickSwitch",
      "ObsidianNew",
      "ObsidianSearch",
      "ObsidianTemplate",
      "ObsidianToday",
      "ObsidianTomorrow",
      "ObsidianYesterday",
      "ObsidianDailies",
      "ObsidianTags",
    },

```

**Comment by josephcrawfordSRH** [comment]
Thank You
